lib/membrane_caps_video_h264.ex

```diff
@@ -56,6 +56,18 @@ defmodule Membrane.Caps.Video.H264 do
   """
   @type alignment_t :: :au | :nal | :none
 
+  @typedoc """
+  When alignment is set to `:au`, determines whether buffers have NALu info attached in metadata.
+
+  If true, each buffer contains the NAL units list under `metadata.h264.nalus`. The list consists of
+  maps with the following entries:
+  - `prefixed_poslen: {pos, len}` - position and length of the NALu within the payload
+  - `unprefixed_poslen: {pos, len}` - as above, but omits Annex B prefix
```

Comment:
maybe `poslen` -> `span` ?

Reply:
I wanted it to conform to https://erlang.org/doc/man/binary.html#part-2